app/src/main/java/ch/epfl/favo/MainActivity.java

```diff
@@ -94,11 +99,35 @@ protected void onCreate(Bundle savedInstanceState) {
     // prevent swipe to open the navigation menu
     drawerLayout.setDrawerLockMode(DrawerLayout.LOCK_MODE_LOCKED_CLOSED);
 
+    // check connection
+    if (DependencyFactory.isOfflineMode(this)) {

```

Comment:
I wonder if DependencyFactory is the right place for this function, maybe it should be stored somewhere else?

Reply:
My bad I see why, nevermind!